common/app/model/content.scala

```diff
@@ -157,6 +158,13 @@ final case class Content(
     else if(isFromTheObserver && tags.isComment) TwitterImage.opinionsObserver
     else if(tags.isComment) TwitterImage.opinions
     else if(tags.isLiveBlog) TwitterImage.live
+    else if(tags.tags.exists(_.id == "tone/news") && trail.webPublicationDate.getYear < DateTime.now().getYear()) {
+      if(isFromTheObserver) {
```

Comment:
Out of interest, what is the behaviour if we think we should have an overlay image, but for whatever reason it is missing. For example somebody decides the s3 bucket should have a retention policy of 30 days and all of the overlay images get deleted out by accident. Will this code 500? Or fallback gracefully.

Reply:
The fallback will be no image overlay at all. But it won't blow up or anything

Reply:
Cool that's fine imo